src/components/astro/AstroClassroomsTable.jsx

```diff
@@ -50,7 +50,7 @@ function AstroClassroomsTable(props) {
           </TableRow>
         </thead>
         {props.classrooms.map((classroom) => {
-          const joinURL = `${window.location.host}/#/${props.selectedProgram.slug}/students/classrooms/${classroom.id}/join?token=${classroom.joinToken}`;
+          const joinURL = `${config.origin}/#/${props.selectedProgram.slug}/students/classrooms/${classroom.id}/join?token=${classroom.joinToken}`;
```

Comment:
I'm running into a bit of WTF-ery here. 90% of the time, `config.origin` lists `localhost:3998` as expected but every once in a while I see weirdness like this:

<img width="753" alt="screen shot 2017-12-01 at 18 27 46" src="https://user-images.githubusercontent.com/13952701/33497299-623f71bc-d6c5-11e7-9d1c-4f6f7ba292a9.png">
_Browser window URL: http://localhost:3998/#/... | Join URL: http://localhost:3998/classroom/#/... | OSX+Chrome+zootester1_

There was a previous case where I'd see the origin listed as `localhost:3998...` instead of `http://localhost:3998...` as well, but as with all matters of WTF-ery, replicating the issue is in itself an issue.

Reply:
_-throws hands up in the air-_ So it looks like my localhost thinks I'm `env=staging` (hence the `/classroom` extension to everything) instead of `env=development`, despite my NODE_ENV being `development`, because... the moon is in the house of Scorpio, who knows. At least I can confine this debug to my machine only and not the code.

Reply:
Maybe an explicit `export NODE_ENV=development` needs to be added to line 10 of the start script in the package.json?

Reply:
OH! No, I think it's actually the `webpack.config` - maybe we need to change this line:
```
//webpack.config.js, line 47
new webpack.DefinePlugin({
  'process.env.NODE_ENV': JSON.stringify('staging')
}),
```